pkg/logs/config/source.go

```diff
@@ -57,3 +61,8 @@ func (s *LogSource) GetInputs() []string {
 	}
 	return inputs
 }
+
+// AddParser add a parser
+func (s *LogSource) AddParser(pm parser.Parser) {
```

Comment:
When I first saw `AddParser` I though this will add a parser to a stack of parsers. Maybe call it `SetParser` instead?